packages/block-editor/src/components/block-list/index.js

```diff
@@ -136,7 +129,16 @@ function BlockList( {
 	);
 }
 
+const ForwardedBlockList = forwardRef( BlockList );
+
 // This component needs to always be synchronous
 // as it's the one changing the async mode
 // depending on the block selection.
-export default forceSyncUpdates( BlockList );
+export default forwardRef( ( props, ref ) => {
+	const fallbackRef = useRef();
```

Comment:
This seems like something that could be built-in `forwardRef` with an option or something.

Reply:
I'd love that. We're doing it in other components. Are we ok with enhancing a React API in `@wordpress/element`?

Reply:
I don't know, maybe we could ask the React team too if this is a pattern they've noticed.

Reply:
Sounds good! Let's discuss separately.